### ISSUE TITLE
on Arch Linux can't find native capsone library

### ISSUE THREAD

**Issue by DeadNumbers** [capstone-rust/capstone-sys#28]
```
error: could not find native static library `capstone`, perhaps an -L flag is missing?

error: aborting due to previous error

error: Could not compile `capstone-sys`.
```

Installed capstone library `community/capstone 3.0.5-1 [installed]`

**Comment by tmfink** [comment]
Since 0.9.0, this crate always builds/users the local captive version on the Capstone C library, so you don't need to install Capstone yourself.

How are you building? What version of capstone-sys are you using? The full terminal output would help.

As a side note, I recommend using [capstone-rs](https://github.com/capstone-rust/capstone-rs) instead this crate directly.

Thanks for reporting the issue.

**Comment by DeadNumbers** [comment]
Hi, @tmfink! I'm try install https://github.com/pkgw/elfx86exts, he has dependencies `capstone = "^0.4"`

Log:
```
==> Starting build()...
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling unicode-xid v0.0.4                                                                                                     
   Compiling quote v0.3.15
   Compiling cfg-if v0.1.5
   Compiling capstone-sys v0.6.0
   Compiling libc v0.2.43
   Compiling plain v0.2.3
   Compiling unicode-width v0.1.5
   Compiling vec_map v0.8.1
   Compiling bitflags v1.0.4
   Compiling ansi_term v0.11.0
   Compiling strsim v0.7.0
   Compiling log v0.4.5
   Compiling synom v0.11.3
   Compiling textwrap v0.10.0
   Compiling atty v0.2.11
   Compiling memmap v0.6.2
   Compiling log v0.3.9
   Compiling syn v0.11.11
   Compiling clap v2.32.0
error: could not find native static library `capstone`, perhaps an -L flag is missing?

error: aborting due to previous error

error: Could not compile `capstone-sys`.
warning: build failed, waiting for other jobs to finish...
error: build failed
==> ERROR: A failure occurred in build().
    Aborting...
==> ERROR: Makepkg was unable to build elfx86exts.
==> Restart building elfx86exts ? [y/N]
==> -----------------------------------
==> 
```

**Comment by waywardmonkeys** [comment]
The `0.4` release of the `capstone` crate is not using the `0.9` release of the `capstone-sys` crate, which lead to this ... There needs to be a new release of `capstone` that points to the newer `capstone-sys` and then a new release of `elfx86exts` would need to be made pointing to the new release of `capstone`.

**Comment by tmfink** [comment]
@waywardmonkeys you beat me to the punch--yes.

**Comment by tmfink** [comment]
I published version `0.9.1` of capstone-sys and `0.5.0` of capstone.

@DeadNumbers You can now open an issue/PR to upgrade the version of capstone that https://github.com/pkgw/elfx86exts/issues uses.
